### DISCUSSION TITLE
FilePicker.get_directory_path keeps closing the interface after directory is selected

### DISCUSSION THREAD

**Discussion by FilipeDVelosa** [flet-dev/flet#1763] [Q&A]
### Question

Hello everyone, I am currently trying to get the path of a certain directory in a simple program that I am creating. In my code, after choosing the desired folder and pressing 'Select Folder' the interface is closed and no error is printed in the Python console. It is even indicated that 'Process finished with exit code 0'. The same does not happen while selecting files. To dismiss any error regarding my code, I used the code in the examples (https://github.com/flet-dev/examples/blob/main/python/controls/file-picker/file-picker-all-modes.py) and I have the same behavior. 

Any idea what might be causing this?

Thanks in advance. 

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by rkleivel** [comment]
Hi!
It is difficult to be conclusive, and you don't say anything about your environment. But I have had issues when running from VSCode in Ubuntu. Check out https://github.com/flet-dev/flet/issues/1530#issuecomment-1639717515 and https://github.com/flet-dev/flet/issues/1530#issuecomment-1641971618, maybe there is something there that can pull you further

**Reply by FilipeDVelosa** [reply]
Sorry, I am running the code in Pycharm, Python 3.11 in a Windows 10 and flet version 0.8.1. I've looked through your answers but could not find anything that would oull me further, but thanks anyway!

**Reply by rkleivel** [reply]
Ok... What happens if you run the code outside pycharm - just 
python yourfile.py
?

**Reply by FilipeDVelosa** [reply]
Running on anaconda terminal I get the same result

**Reply by ndonkoHenri** [reply]
Can you make a video showing the error? 
I just tried the code with win10, and it worked as expected. 

![image](https://github.com/flet-dev/flet/assets/98978078/22802af0-a167-4670-9e6e-4e9333c2ef31)

**Comment by CoutinhoElias** [comment]
Just taking advantage of the context:

How can I use the chosen directory to fetch files from this directory?

My difficulty is using the chosen directory path. Normally I receive None, although it works in the same way as you see in the image in the previous answer.
